content/3.Joining Corda Network/5.Offboarding Workflow.md

```diff
@@ -0,0 +1,22 @@
+If you foresee that one or some of your Corda Network network certificate(s) will not be used permanently or will not re-join in the long-run, you may submit a request to revoke the certificate.
```

Comment:
Maybe worth distributing the page into 3 sections:

1. Temporary Suspension from Corda Network

And copy this text here: If you do not intend for the network certificate to be dormant permanently but want to hide the node from the network map, you may shut down the node and it will be hidden from the network map, and once you restart the node it will show up on the network map automatically. You can read more about this feature *(**eventHorizon**)* on the [Corda Documentation Site](https://docs.corda.net/docs/corda-enterprise/4.8/network/network-reqs.html#corda-network-parameters).

2. Revocation of node(s) 
If you foresee that one or some of your Corda Network network certificate(s) will not be used permanently or will not re-join in the long-run, you may submit a request to revoke the certificate.

This text is then followed by detailed instructions on how to submit a ticket

3. Terminating Corda Network contract

Reply:
Updated to this format :)